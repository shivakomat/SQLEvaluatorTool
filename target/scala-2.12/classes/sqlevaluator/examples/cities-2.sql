-- Capital cities, excluding India
SELECT cities.name AS name, country, population
FROM countries, cities
WHERE capital = cities.name AND country != "India"
