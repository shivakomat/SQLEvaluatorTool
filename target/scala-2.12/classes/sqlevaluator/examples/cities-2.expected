[
    [["name", "str"], ["country", "str"], ["population", "int"]],
    ["Washington DC", "USA", 681],
    ["Ottawa", "Canada", 934],
    ["Tokyo", "Japan", 13513]
]
